hibernate-core/src/main/java/org/hibernate/engine/spi/Mapping.java

```diff
@@ -23,10 +24,11 @@
  * @deprecated Use {@link org.hibernate.type.spi.TypeConfiguration},
  * {@link org.hibernate.boot.Metadata}, or
  * {@link org.hibernate.metamodel.RuntimeMetamodels}
+ * or {@link  org.hibernate.type.spi.MappingContext}
  * to access such information
  */
 @Deprecated(since = "6.0")
-public interface Mapping {
+public interface Mapping extends MappingContext {
 	Type getIdentifierType(String className) throws MappingException;
 	String getIdentifierPropertyName(String className) throws MappingException;
 	Type getReferencedPropertyType(String className, String propertyName) throws MappingException;
```

Comment:
I think we can just delete this methods and leave the interface empty, right?

Reply:
This would unfortunately be a binary incompatible change, though maybe that is ok since this is an SPI?

Reply:
I don't believe we guarantee binary compatibility between major versions.